W-16794040 updates to outlook config (#426)
mulesoft/docs-rpa: rpa-builder/6.7/modules/ROOT/pages/toolbox-mail-operations-mail-session-oauth.adoc

== Before You Begin

> Before you configure this action step, ensure you have the following based on the mail provider you use:

Author: ```suggestion
Depending on the mail provider you use, ensure you have the following:
```

* *Outlook*
** An Azure account

> ** SMTP AUTH enabled for your Microsoft organization or the specific mail when connecting with an Azure environment via SMTP

Reviewer: **Question**: Shouldn't this say "... or the specific mail**box** when connecting ..."?

> ** SMTP AUTH enabled for your Microsoft organization or the specific mail when connecting with an Azure environment via SMTP

Author: ```suggestion
** SMTP AUTH enabled for your Microsoft organization or the specific mailbox when connecting with an Azure environment via SMTP
```
